Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -69,6 +73,9 @@ open class ChatMessageContentView: _View, ThemeProvider {
 
     /// The delegate responsible for action handling.
     public weak var delegate: ChatMessageContentViewDelegate?
+    
+    /// The delegate responsible for interactions with mentions.
+    public weak var mentionDelegate: ChatMessageMentionedUserDelegate?
```

Comment:
Is there any reason why we didn't use the `ChatMessageContentViewDelegate`? IMO it makes sense to keep things consistent for now and move this to the `ChatMessageContentViewDelegate`

Reply:
I used a separated delegate because since we also have to provide a similar implementation in the ComposerVC, the ComposerVC would have to conform to the `ChatMessageContentViewDelegate` too, which is unrelated to the ComposerVC, and would have to implement all of the current methods on that protocol which we would have to leave empty.

Reply:
I get the idea, but we are not using the `ChatMessageMentionedUserDelegate` in the `ComposerVC`. We conform to it, but the delegate is never called, we just call directly a function which is part of the `ComposerVC`, that is not how the delegate pattern should be used, the delegate pattern is about delegating responsibility to another component. Here, we are delegating the responsibility to the same component, which does not make sense. Besides that, it is very unlikely that we need to handle clicking on the composer. IMO there's no need to reuse the same delegate in both places (we are not doing it right now either way).